task/github-add-comment/0.7/tests/fixtures/list-comment-response.json

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "url": "https://api.github.com/repos/tektoncd/plumbing/issues/comments/111111111111",
+    "html_url": "https://github.com/tektoncd/plumbing/pull/111#issuecomment-111111111111",
+    "issue_url": "https://api.github.com/repos/tektoncd/plumbing/issues/111",
+    "id": 111111111111,
+    "node_id": "abcd==",
```

Comment:
same as above, do we still need this file?

Reply:
see other comment :)